### DISCUSSION TITLE
Proxy to code-server via path on webserver

### DISCUSSION THREAD

**Discussion by ubarilan** [coder/code-server#5452] [Q&A]
Hello, here's what I wanna do:
I want `https://my-web-server/some_path` to have the code-server on it.
Are there any options I need to use on code-server in order for this to work perfectly?

**Comment by jsjoeio** [comment]
Hmm...sounds like you want a reverse proxy? You could look into using [Caddy](https://coder.com/docs/code-server/latest/guide#using-lets-encrypt-with-caddy). Our docs should suffice. If not, let us know!

**Reply by ubarilan** [reply]
Unfortunately it does not solve my use case as I need to create proxies at runtime with my NodeJS app.

**Reply by jsjoeio** [reply]
Ah okay. Well I don't know how much help I can be then. Best advice is to try it and see what works. If you need help after, come back and we'll see what we can do!
